ci/Dockerfile.ubuntu

```diff
@@ -0,0 +1,59 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG DISTRO_VERSION=17.04
+FROM ubuntu:${DISTRO_VERSION}
+MAINTAINER "Carlos O'Ryan <coryan@google.com>"
+
+RUN apt-get update
+RUN apt-get install -y \
+  build-essential \
+  git \
+  gcc \
+  g++ \
+  golang \
+  clang \
+  cmake \
+  make \
+  tar \
+  wget
+
+RUN if grep -q 14.04 /etc/lsb-release; then apt-get install -y cmake3; fi
+# Install optional packages that probably fail in 14.04, but we only use in other distributions anyway ...
```

Comment:
Are you saying that these deps are **required** for post-14.04 versions, but are **optional** for the 14.04 version, and the build + test still succeed? That's interesting.

Reply:
PTAL.